common/templates/general.go

```diff
@@ -255,3 +255,52 @@ func tmplFormatTime(t time.Time, args ...string) string {
 
 	return t.Format(layout)
 }
+
+// slice returns the result of creating a new slice with the given arguments.
+// "slice x 1 2" is, in Go syntax, x[1:2], and "slice x 1" is equivalent to
+// x[1:].
+func slice(item reflect.Value, indices ...reflect.Value) (reflect.Value, error) {
+	v, _ := indirect(item)
+	if !v.IsValid() {
+		return reflect.Value{}, errors.New("index of untyped nil")
+	}
+
+	var args []int
```

Comment:
Hmm feel like this is a bit sketchy, but atleast make it []int64 since that's used more and will stop unexpected problems from happening on 32bit systems.

Reply:
I really hope there isn't a scenario where someone needs a 64-bit slice. 😄 I chose int rather than int64 because reflect.Value.{Len,Slice}() use an int and so I a) end up with fewer type conversions and b) am not pretending to support ranges that aren't true.

I'm glad to switch if you still wish, but I don't see the specific architecture concerns.

I didn't intend this to be "sketchy".

Reply:
All discord id's are 64bit so pretty commonly used

Reply:
Ahhh yeah i see i misread it, woops, sorry man